notifications_utils/polygons.py

```diff
@@ -82,7 +82,7 @@ def buffer_outward_in_degrees(self):
         '''
         return (
             # If two areas are close enough that the distance between
-            # them is less than the minimum bleed of a cell
+            # them is less than the typical bleed of a cell
```

Comment:
Optional: I wonder if the property name `approx_bleed_in_degrees` would now be more appropriate as something like `default_bleed_in_degrees` and if there is any updating we want to do of the description above that property? I don't feel too strongly though